playbooks/template_pnp_intent.yml

```diff
@@ -20,16 +20,15 @@
         state: "merged"
         #ignore_errors: true        #Enable this to continue execution even the task fails
         config:
-        - configuration_templates:
```

Comment:
In the latest code, I could see configuration_templates.. I think we don't need to do changes in template_intent playbook. Can you please check and confirm?

Reply:
Last time when we worked on the demo playbook, we didn't need any configuration part?

Reply:
Can you check the latest code? I think you dont need any changes in template intent part.